bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -15,6 +15,12 @@ namespace python {
 void exposeContact1D() {
   bp::register_ptr_to_python<boost::shared_ptr<ContactModel1D> >();
 
+  bp::enum_<pinocchio::ReferenceFrame>("ReferenceFrame")
```

Comment:
You should not duplicate this. This is a Pinocchio feature.

Reply:
If you want to have this thing exposed in Crocoddyl, there is a specific helper function to import ReferenceFrame within the Croco namespace, assuming Pinocchio is exposed first.

Reply:
Let's do not expose this in Crocoddyl.
There is no reason to duplicate this code.

Reply:
[fd58261](https://github.com/loco-3d/crocoddyl/pull/1054/commits/fd582614a34b626d8cb0751df8132d31f643845b)